presto-native-execution/presto_cpp/main/connectors/PrestoToVeloxConnector.cpp

```diff
@@ -1004,6 +984,26 @@ connector::hive::HiveColumnHandle::ColumnType toHiveColumnType(
   }
 }
 
+velox::common::CompressionKind toFileCompressionKind(
```

Comment:
Why move this function to between the two toHiveColumnType()? If not particular reason let's move it back.